src/request/sign-swap/SignSwap.js

```diff
@@ -56,26 +62,36 @@ class SignSwap {
         /** @type {HTMLDivElement} */
         const $exchangeRate = (this.$el.querySelector('#exchange-rate'));
         /** @type {HTMLDivElement} */
+        const $rightAccount = (this.$el.querySelector('.right-account'));
+        /** @type {HTMLDivElement} */
+        const $leftAccount = (this.$el.querySelector('.left-account'));
+        /** @type {HTMLDivElement} */
         const $leftIdenticon = (this.$el.querySelector('.left-account .identicon'));
         /** @type {HTMLDivElement} */
         const $rightIdenticon = (this.$el.querySelector('.right-account .identicon'));
         /** @type {HTMLSpanElement} */
         const $leftLabel = (this.$el.querySelector('.left-account .label'));
         /** @type {HTMLSpanElement} */
+        const $leftNewBalance = (this.$el.querySelector('.left-account .new-balance'));
+        /** @type {HTMLSpanElement} */
         const $rightLabel = (this.$el.querySelector('.right-account .label'));
         /** @type {HTMLDivElement} */
+        const $rightNewBalance = (this.$el.querySelector('.right-account .new-balance'));
```

Comment:
```suggestion
        const $rightNewBalance = ($rightAccount.querySelector('.new-balance'));
```
And similarly for the others.